ts/Stock/Indicators/IKH/IKHIndicator.ts

```diff
@@ -0,0 +1,960 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+*
+* Import
+*
+* */
+
+import type ColorType from '../../../Core/Color/ColorType';
+import type IndicatorValuesObject from '../IndicatorValuesObject';
+import type LineSeries from '../../../Series/Line/LineSeries';
+import type Point from '../../../Core/Series/Point';
+import type IKHPoint from './IKHPoint';
+import type SVGElement from '../../../Core/Renderer/SVG/SVGElement';
+import type SVGPath from '../../../Core/Renderer/SVG/SVGPath';
+import BaseSeries from '../../../Core/Series/Series.js';
+const {
+    seriesTypes: { sma: SMAIndicator }
+} = BaseSeries;
+import Color from '../../../Core/Color/Color.js';
+const color = Color.parse;
+import H from '../../../Core/Globals.js';
+import U from '../../../Core/Utilities.js';
+import {
+    IKHDrawSenkouSpanObject,
+    IKHGapExtensionObject,
+    IKHOptions,
+    IKHParamsOptions,
+    IKHSenkouSpanOptions
+} from './IKHOptions';
```

Comment:
The import should make it clear that it is about types. Therefor explicitly do `import type`.

Reply:
Added 'type' import